sc_az_reindex_dag.py

```diff
@@ -51,26 +67,84 @@
 Tasks with custom logic are relegated to individual Python files.
 """
 
-GET_NUM_SOLR_DOCS_PRE = task_solrgetnumdocs(AZ_DAG, CONFIGSET, 'get_num_solr_docs_pre', conn_id=SOLR_CONN.conn_id)
+# Single task to provide message about which controller triggered the DAG
+REMOTE_TRIGGER_MESSAGE = BashOperator(
+    task_id="remote_trigger_message",
+    bash_command='echo "Remote trigger: \'{{ dag_run.conf["message"] }}\'"',
+    dag=DAG,
+)
 
-CREATE_COLLECTION = create_sc_collection(AZ_DAG, SOLR_CONN.conn_id, COLLECTION, REPLICATION_FACTOR, CONFIGSET)
+REQUIRE_DAG_RUN = PythonOperator(
+    task_id="require_dag_run",
+    python_callable=require_dag_run,
+    provide_context=True,
+    dag=DAG,
+)
+
+SET_COLLECTION_NAME = PythonOperator(
+    task_id="set_collection_name",
+    python_callable=datetime.now().strftime,
+    op_args=["%Y-%m-%d_%H-%M-%S"],
+    dag=DAG
+)
+
+GET_NUM_SOLR_DOCS_PRE = task_solrgetnumdocs(
+    DAG,
+    CONFIGSET,
```

Comment:
I  like the way that we pass configset from the trigger tag in the funcake_dags trigger here: https://github.com/tulibraries/funcake_dags/blob/b58bf0c9a15d77424898545aa7f906062d0724d9/trigger_dev_funcake_index_dag.py#L35
Is that not a good option?

Reply:
Why would you parametrize it? Adding that extra parameter does nothing.

Reply:
Because at least in funcake_dags we were calling these DEV/PROD* I was hoping we could parameterize all the variables that vary between the dev and prod and whatnot run to explicitly make that clear from the trigger dag.